tests/client_tests.ts

```diff
@@ -161,113 +157,83 @@ describe.each([
-          expect(response.process).toHaveProperty('memory', expect.any(Number))
-          expect(response.process).toHaveProperty('cpu', expect.any(Number))
-        })
-      })
-      test(`${permission} key: get pretty system info`, async () => {
-        await client.prettySysInfo().then((response: Types.SysInfoPretty) => {
-          expect(response).toHaveProperty('memoryUsage', expect.any(String))
-          expect(response).toHaveProperty('processorUsage', expect.any(Array))
-          expect(response.global).toHaveProperty(
-            'totalMemory',
-            expect.any(String)
-          )
-          expect(response.global).toHaveProperty(
-            'usedMemory',
-            expect.any(String)
-          )
-          expect(response.global).toHaveProperty(
-            'totalSwap',
-            expect.any(String)
-          )
-          expect(response.global).toHaveProperty('usedSwap', expect.any(String))
-          expect(response.global).toHaveProperty(
-            'inputData',
-            expect.any(String)
-          )
-          expect(response.global).toHaveProperty(
-            'outputData',
-            expect.any(String)
-          )
-          expect(response.process).toHaveProperty('memory', expect.any(String))
-          expect(response.process).toHaveProperty('cpu', expect.any(String))
-        })
-      })
-      test(`${permission} key: get /stats information`, async () => {
-        await client.stats().then((response: Types.Stats) => {
-          expect(response).toHaveProperty('databaseSize', expect.any(Number))
-          expect(response).toHaveProperty('lastUpdate') // TODO: Could be null, find out why
-          expect(response).toHaveProperty('indexes', expect.any(Object))
-        })
-      })
-    })
+    describe('Test on base routes', () => {})
   }
 )
 
 describe.each([{ client: privateClient, permission: 'Private' }])(
   'Test on routes where private key should not have access',
   ({ client, permission }) => {
     describe('Test on base routes', () => {
-      test(`${permission} key: try to get version and be denied`, async () => {
-        await expect(client.version()).rejects.toThrowError(
-          `Invalid API key: ${PRIVATE_KEY}`
-        )
-      })
-      test(`${permission} key: try to get system info and be denied`, async () => {
-        await expect(client.sysInfo()).rejects.toThrowError(
-          `Invalid API key: ${PRIVATE_KEY}`
-        )
-      })
-      test(`${permission} key: try to get pretty system info and be denied`, async () => {
-        await expect(client.prettySysInfo()).rejects.toThrowError(
-          `Invalid API key: ${PRIVATE_KEY}`
-        )
-      })
-      test(`${permission} key: try to get /stats information and be denied`, async () => {
-        await expect(client.stats()).rejects.toThrowError(
-          `Invalid API key: ${PRIVATE_KEY}`
-        )
-      })
+      // test(`${permission} key: try to get system info and be denied`, async () => {
```

Comment:
Why is it commented?

Reply:
forgot to delete it.! Thanks

Reply:
For the same reason you explained in the other comment of my review? 🙂

Reply:
Yes !